resources/schema.json

```diff
@@ -307,15 +307,33 @@
                     {
                       "required": [
                         "variables"
-                      ]
+                      ],
+                      "not":
+                      {
+                        "required": [
+                          "target"
+                        ]
+                      }
+                    },
+                    {
+                      "required": [
+                        "target"
+                      ],
+                      "not":
+                      {
+                        "required": [
+                          "variables"
+                        ]
+                      }
```

Comment:
Can you quickly explain, what these `not` clauses are about? Is it to indicate that, e.g., when `target` is given, `variables` don't have to? Wouldn't it be the same then with the existing rule? 

```js
"oneOf": [
  {
    "required": [
      "variables"
    ]
  },
  {
    "required": [
      "target"
    ]
  }
]
```

From my understanding, this already makes it possible to have both available. It only says, that at least one is required. Or do I miss something?

Reply:
In this occasion:

* As discussed, oneOf is exclusive
* The `not`s make the rules explicit and mutually exclusive
* Alternatively we could work with `anyOf` but I like the explicitness of the rule

Reply:
Philipp also gave feedback here (https://github.com/camunda/element-templates-json-schema/pull/30#discussion_r661458029). I used `anyOf` now, it has the same result and requires a lot less config.